ONSdigital/ssdc-rm-caseprocessor
# Pull release artifacts from release repo

# Motivation and Context
We need to be able to pull RELEASE artifacts from the releases repo in ons-ci-rm

# What has changed
Added repo details for RELEASE artifacts

# How to test?
- Amend the pom to pull a RELEASE version of shared-sample-validation or common-entity-model (ensure the release artifacts have been created first).
- Run a build and check that it can pull in the correct artifacts

# Links
https://trello.com/c/d7cWeH4n
